src/extension.ts

```diff
@@ -233,6 +233,7 @@ function startServerFor(folder: WorkspaceFolder) {
     languageClient.start();
 
     languageClient.onReady().then(() => {setLanguageId(languageClient, folder.uri.fsPath); });
+    registerFileWatcher(folder.uri.fsPath);
```

Comment:
can we move this to a different file? `extension.ts` needs to be debloated already!